crates/nu-cmd-extra/src/extra/bits/shift_left.rs

```diff
@@ -90,12 +90,12 @@ impl Command for BitsShl {
             },
             Example {
                 description: "Shift left a number with 1 byte by 7 bits",
-                example: "2 | bits shl 7 -n '1'",
+                example: "2 | bits shl 7 --number-bytes '1'",
```

Comment:
not related to this PR, but I wonder why `--number-bytes` takes a string. Looks odd.

Reply:
Seems like this command was introduced by @hustcer with https://github.com/nushell/nushell/pull/6202.
Maybe they remember if there was a reason for this or if this is an oversight :)

Reply:
> not related to this PR, but I wonder why `--number-bytes` takes a string. Looks odd.

Actually `--number-bytes` takes a int from the beginning, but changed to a string in 90b65018b

Reply:
I might be confused here, but doesn't this mean it took a string from the very beginning?
https://github.com/hustcer/nushell/blob/45897db03fa4d5e0956e4b0038015a61d3299df1/crates/nu-command/src/bits/shift_left.rs#L29

Reply:
Oh, I see, "it can be 1, 2, 4, 8, auto", `auto` is a string

Reply:
Ah yeah :)